### ISSUE TITLE
[BUG] - invalid literal for int() with base 10: 'None'

### ISSUE THREAD

**Issue by RikkB** [receyuki/comfyui-prompt-reader-node#36]
### Description

I read one previous report with this issue, but op said he had complicated workflow.  Also response suggested using Prompt Saver to save metadata w images.
I'm trying to read metadata from previously generated (ComfyUI) png images.  The only node I have in my workspace is the Prompt Reader yet still get this error.

### Reproduction steps

Start with clean, clear workspace
Load only the Prompt Reader node
Load image, either from slider or by upload 
Queue Prompt
error generated

### Image file

![workflow](https://github.com/receyuki/comfyui-prompt-reader-node/assets/154370212/7d2761c6-15f0-4093-9e7c-aea90dfea39a)

**Comment by RikkB** [comment]
Error occurred when executing SDPromptReader:

invalid literal for int() with base 10: 'None'

File "C:\Users\rickb\ComfyUI_windows_portable\ComfyUI\execution.py", line 153, in recursive_execute
output_data, output_ui = get_output_data(obj, input_data_all)
File "C:\Users\rickb\ComfyUI_windows_portable\ComfyUI\execution.py", line 83, in get_output_data
return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
File "C:\Users\rickb\ComfyUI_windows_portable\ComfyUI\execution.py", line 76, in map_node_over_list
results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
File "C:\Users\rickb\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-prompt-reader-node\nodes.py", line 161, in load_image
seed = int(

**Comment by receyuki** [comment]
Can you post the original image here?

**Comment by SwannSchilling** [comment]
Same here!

**Comment by RikkB** [comment]
> Can you post the original image here?

![ComfyUI_temp_ipzza_00066_](https://github.com/receyuki/comfyui-prompt-reader-node/assets/154370212/18e7d1bc-c4b7-4ef8-901c-dcba957129e5)
Since  I'm pretty new to all this, am wondering if the img was an SDXL generation and that's why it didn't work?  Quite a bit of my work errors out, but haven't yet gotten organized enough to have separated the SD1.5 from the XL stuff early on.  The reader does work on some of my images.

**Comment by SwannSchilling** [comment]
There was no image being generated in my case because of the error. But it was SDXL also...

**Comment by receyuki** [comment]
@RikkB @SwannSchilling 
The workflow in the image above uses Efficiency Nodes. Currently, the SD Prompt Reader does not support the Efficiency Node. In fact, this error message is due to the lack of support for custom nodes, not just the Efficiency Node.

The reason for this is quite simple: for each custom node, I have to support them individually. Although I have supported a large number of custom nodes, this is still an endless task. That's why I decided to create Prompt Saver node.
